engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -23,6 +20,8 @@ import scala.concurrent.{Await, Promise}
 import scala.language.postfixOps
 
 object WorkflowManagerActor {
+  val DefaultMaxWorkflowsToLaunch = 50
+  val DefaultNewWorkflowPollRate = 20
```

Comment:
I actually think I prefer this as a push rather than a pull, since it means the WMA doesn't need to change at all, and you can have different clients behave differently (e.g. the single runner could submit instantly, whereas the server could delay and submit in batches)